dojo/db_migrations/0072_remove_system_settings_s_finding_severity_naming.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.17 on 2021-01-30 22:53
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0071_product_type_enhancement'),
```

Comment:
there is a need to change that as we merged some PR with migration (see the current number in dev branch)